### ISSUE TITLE
Decide on theme of splash screen

### ISSUE THREAD

**Issue by circuitsunfish** [circuitsunfish-hutch86-biljanakotev/splash-screen#1]
CryptoJoggler and I were discussing themes and we both like the idea of classic video games from the arcade cabinet era - Pac-Man, Galaga, Space Invaders; etc. 

Before we can build things we need to settle on what we will build.

**Comment by MaudHutch86** [comment]
Good point circuitsalsa ! Personally , I think TRON or Donkey are two  good option we could play with ? 
TRON because we do not need heavy gears to have a descent effect.
Donkey kong or alike game because we could basically create something fun ( donkey climbing the pyramid and get the girl on top when page will be fully loaded ) !
What do you think guys ?

**Comment by circuitsunfish** [comment]
Massive fan of TRON myself (used to spend tons of hours in armacycles / armagatronad)

**Comment by BiljanaKotev** [comment]
Love the donkey kong idea maybe once he reaches the top and gets the girl the image fades and the words donkey kong show up in somkind of animated way?
What library would we use for this type of animation?

**Comment by MaudHutch86** [comment]
Donkey kong could match with the animal theme @circuitsunfish ! But it is much more technical than the Tron idea ... So now we need to choose 🙃

**Comment by MaudHutch86** [comment]
@BiljanaKotev gsap.js could be a good library

**Comment by circuitsunfish** [comment]
@MaudHutch86 what if Donkey Kong is riding a troncycle? we/I could probably just drop a DK sprite on top of an armagatronad cycle in GIMP. Could be pretty funny and has that Wreck-it-Ralph / Arcade crossover element

**Comment by MaudHutch86** [comment]
@BiljanaKotev  should we follow @circuitsunfish  last idea ?

**Comment by BiljanaKotev** [comment]
Yea i like that let’s try it, i think it’s dynamic but still simple enough to implement .

**Comment by BiljanaKotev** [comment]
@MaudHutch86 @circuitsunfish 
I've been playing around with anime.js a bit and came up with something. It's just for fun and for me to practice but maybe we can take some bits from it?

https://codepen.io/BibaKotev/full/dyqOpKx

**Comment by MaudHutch86** [comment]
So we finally decided what will the splash screen like .

We will use the "spy in disguise " splash screen as a inspiration but instead we will have Donkey Kong running then jump in the air and confetti will appear as he is up with maybe some celebration  text.
